composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "This is a Micro plugin for webpack-encore support",
     "license": "MIT",
     "type": "micro-plugin",
-    "version": "1.1",
+    "version": "1.2",
```

Comment:
composer verify --string failed.

Reply:
@Asisyas, version removed, added `composer validate --strict` to static GitHub actions.